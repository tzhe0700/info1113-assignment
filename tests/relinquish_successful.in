ADD STUDENT Harry Potter
ADD SPELLBOOK spellbooks.csv 111111
RENT 100000 111111
RELINQUISH 100000 111111
EXIT
