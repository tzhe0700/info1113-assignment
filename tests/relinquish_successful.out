user: Success.

user: Successfully added: Advanced Potion Making (Libatius Borage).

user: Success.

user: Success.

user: Ending Archive process.
